src/app.ts

```diff
@@ -2,11 +2,24 @@ import express from 'express';
 
 import config from 'config';
 import routes from './routes/ninja';
+import { dbCheck } from './db/database';
 
 const app = express();
 const port: number = config.get('App.port');
+const host: string = config.get('App.host');
 app.use(routes);
 
-app.listen(port, () => {
-  console.log(`Server is listening on port ${port}...`);
-});
+async function startServer() {
+  try {
+    const dbCheckResult = await dbCheck();
+    console.log(dbCheckResult);
```

Comment:
Logs should comes from logger client. For instance you can use bunyan library. It providing for example logging levels.